gtk/transmission.gresource.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <gresources>
   <gresource prefix="/com/transmissionbt/transmission">
-    <file alias="icons/scalable/actions/lock.svg">icons/lock.svg</file>
-    <file alias="icons/scalable/actions/options-symbolic.svg">icons/options-symbolic.svg</file>
-    <file alias="icons/scalable/actions/ratio-symbolic.svg">icons/ratio-symbolic.svg</file>
-    <file alias="icons/scalable/actions/turtle-symbolic.svg">icons/turtle-symbolic.svg</file>
-    <file alias="icons/scalable/apps/transmission.svg" compressed="true" preprocess="xml-stripblanks">icons/hicolor_apps_scalable_transmission.svg</file>
+    <file alias="icons/scalable/actions/lock.svg">../icons/lock.svg</file>
+    <file alias="icons/scalable/actions/options-symbolic.svg">../icons/options-symbolic.svg</file>
+    <file alias="icons/scalable/actions/ratio-symbolic.svg">../icons/ratio-symbolic.svg</file>
+    <file alias="icons/scalable/actions/turtle-symbolic.svg">../icons/turtle-silhouette.svg</file>
```

Comment:
*(nit)* Maybe it's a good idea to change `turtle-symbolic.svg` to `turtle-silhouette.svg` here and in GTK code.

Reply:
GTK expects this particular naming for icons to be in fact considered symbolic, see https://gitlab.gnome.org/GNOME/gtk/-/blob/main/gtk/gtkicontheme.c

Reply:
Is there anything further to do here for me, or is this considered resolved?

Reply:
I consider it resolved, I just don't have the "Resolve" button to click. 😄